src/EnergyPlus/PVWatts.cc

```diff
@@ -56,14 +56,21 @@
 #include <EnergyPlus/DataEnvironment.hh>
 #include <EnergyPlus/DataHVACGlobals.hh>
 #include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/ElectricPowerServiceManager.hh>
```

Comment:
A lot of the action is in this file. Here I change the calls to PVWatts to reference the SSC library rather than code I hijacked from it a few years ago. More maintainable that way.